tests/EmptyBasicFieldsTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+use Corcel\Acf\Field\Text;
+use Corcel\Acf\Field\BasicField;
+use Corcel\Post;
+
+/**
+ * Class BasicFieldTest.
+ *
+ * @author Junior Grossi <juniorgro@gmail.com>
+ */
+class EmptyBasicFieldsTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Post
+     */
+    protected $post;
+
+    /**
+     * Setup a base $this->post object to represent the page with the basic fields.
+     */
+    protected function setUp()
+    {
+        $this->post = Post::find(82); // it' a page with empty custom fields
+    }
+
+    public function testTextFieldValue()
+    {
+        $text = new Text();
+        $text->process('fake_text', $this->post);
```

Comment:
Ah sure - I must have been working off the wrong branch locally then - I swore I was on dev. I will have another look tomorrow and resubmit the PR on the right branch.